docs/filter-functions.md

```diff
@@ -406,6 +408,39 @@ automations:
           reviewers: {{ repo | codeExperts(gt=10) }}
 ```
 
+#### `decode`
+
+Decode base64 string into and object
+
+<div class="filter-details" markdown=1>
+
+| Argument | Usage  | Type   | Description                         |
+| -------- | ------ | ------ | ----------------------------------- |
+| -        | Input  | String | Base64 representation of the object |
+| -        | Output | Object | Decoded objet                       |
+
+</div>
+
+```yaml+jinja
+{{ base64String | decode }}
+```
+
+#### `encode`
+
+Encode data into base64 representation. When an encoded string is passed as input for [`add-comment`](./automation-actions.md#add-comment), the action automatically detects and decodes it.
+<div class="filter-details" markdown=1>
+
+| Argument | Usage  | Type         | Description                         |
+| -------- | ------ | ------------ | ----------------------------------- |
+| -        | Input  | Object       | The input object to encode          |
+| -        | Output | String (b64) | Base 64representation of the object |
+
+</div>
+
+```yaml+jinja
+{{ "String" | encode }}
```

Comment:
I would add a meaningful usage as example as this one doesnt say nothing on how to use it, it could be any filter...

Reply:
Added better example